src/mjolnir/restrictionbuilder.cc

```diff
@@ -270,8 +270,8 @@ ComplexRestrictionBuilder CreateComplexRestriction(const OSMRestriction& restric
 };
 
 struct Result {
-  uint32_t forward_restrictions_count;
-  uint32_t reverse_restrictions_count;
+  uint32_t forward_restrictions_count = 0;
```

Comment:
this finally fixes an issue where we report ridiculous numbers for restrictions, like 2 billion restrictions on level 2